pymc3/step_methods/metropolis.py

```diff
@@ -94,6 +98,28 @@ def __call__(self, num_draws=None):
             return np.dot(self.chol, b)
 
 
+class SettingNotFoundInAttribute(BaseException):
+    pass
+
+
+class SettingsResetter:
+    """Stores a copy of initial settings so they can be reset on call"""
+
+    initial_settings: dict[str, Any]
```

Comment:
```suggestion
    initial_settings: Dict[str, Any]
```
To fix the docs build.